build.gradle

```diff
@@ -141,92 +142,79 @@ task testExternalApis(type: Test) {
 }
 
 wrapper {
-    gradleVersion = '4.8.1'
+    gradleVersion = '5.2.1'
 }
 
-// This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
-if (JavaVersion.current().isJava8Compatible()) {
-    tasks.withType(Javadoc) {
-        options.addStringOption('Xdoclint:none', '-quiet')
-    }
+tasks.withType(Javadoc) {
+    options.addStringOption('Xdoclint:none', '-quiet')
 }
 
 task javadocJar(type: Jar, dependsOn: javadoc) {
-    classifier = 'javadoc'
+    archiveClassifier.set('javadoc')
     from 'build/docs/javadoc'
 }
 
 task sourcesJar(type: Jar) {
     from sourceSets.main.allSource
-    classifier = 'sources'
-}
-
-/**
- * This specifies what artifacts will be built and uploaded when performing a maven upload.
- */
-artifacts {
-    archives jar
-    archives javadocJar
-    archives sourcesJar
+    archiveClassifier.set('sources')
 }
 
 /**
  * Sign non-snapshot releases with our secret key.  This should never need to be invoked directly.
  */
 signing {
-    required { isRelease && gradle.taskGraph.hasTask("uploadArchives") }
+    required { isRelease && gradle.taskGraph.hasTask("publishHtsjdkPublicationToMavenRepository") }
```

Comment:
Is the uppercase 'H' in "Htsjdk" significant? It seems to work ok locally (it winds up in "htsjdk"ok).

Reply:
I don't believe it's meaningful.  I think it's case insensitive but the examples on the gradle website use camel case so I did as well.